src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -586,22 +587,15 @@ private void CheckDisposed()
 
         private static bool IsWebSocketsSupported()
         {
-#if NETCOREAPP3_0
-            // .NET Core 2.1 and above has a managed implementation
-            return true;
-#else
-            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-            if (!isWindows)
+            try
```

Comment:
I don't think we can do a compile time check here, for example, someone running on netcoreapp3.0 but targeting the netstandard2.0 library on windows 7 will have access to the managed websockets impl, but if we just hardcoded netstandard2.0 as not having websockets then a scenario that should work will not work.

Reply:
This kinda sucks....